auth0_flutter/README.md

```diff
@@ -293,36 +322,117 @@ You can make users land directly on the Signup page instead of the Login page by
+
+You can configure the ID token validation by passing an `IdTokenValidationConfig` instance. Check the [API documentation](https://pub.dev/documentation/auth0_flutter_platform_interface/latest/auth0_flutter_platform_interface/IdTokenValidationConfig-class.html) to learn more about the available configuration options.
 
-auth0_flutter automatically [validates](https://auth0.com/docs/secure/tokens/id-tokens/validate-id-tokens) the ID Token obtained from Web Auth login, following the [OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html). This ensures the contents of the ID Token have not been tampered with and can be safely used.
+```dart
+const config = IdTokenValidationConfig(leeway: 10);
+final credentials =
+    await auth0.webAuthentication().login(idTokenValidationConfig: config);
+```
 
 ##### Custom Domains
 
-Users of Auth0 Private Cloud with Custom Domains still on the [legacy behavior](https://auth0.com/docs/deploy/private-cloud/private-cloud-migrations/migrate-private-cloud-custom-domains) need to specify a custom issuer to match the Auth0 Domain when performing Web Auth login. Otherwise, the ID Token validation will fail.
+Users of Auth0 Private Cloud with Custom Domains still on the [legacy behavior](https://auth0.com/docs/deploy/private-cloud/private-cloud-migrations/migrate-private-cloud-custom-domains) need to specify a custom issuer to match the Auth0 Domain when performing Web Auth login. Otherwise, the ID token validation will fail.
 
 ```dart
-final config = IdTokenValidationConfig(issuer: 'https://YOUR_AUTH0_DOMAIN/');
-final result =
-    await auth0.webAuthentication.login(idTokenValidationConfig: config);
+const config =
+    IdTokenValidationConfig(issuer: 'https://YOUR_AUTH0_DOMAIN/');
+final credentials =
+    await auth0.webAuthentication().login(idTokenValidationConfig: config);
 ```
 
 #### Web Auth errors
 
-Web Auth will only throw `WebAuthException` exceptions.<!-- Check the API documentation [link to API documentation] to learn more about the available `WebAuthException` properties. -->
+Web Auth will only throw `WebAuthException` exceptions. Check the [API documentation](https://pub.dev/documentation/auth0_flutter_platform_interface/latest/auth0_flutter_platform_interface/WebAuthException-class.html) to learn more about the available `WebAuthException` properties.
 
 ```dart
 try {
-  final result = await auth0.webAuthentication.login();
+  final credentials = await auth0.webAuthentication().login();
   // ...
 } on WebAuthException catch (e) {
-  print(e.toString())
+  print(e.toString());
+}
+```
+
+### Credentials Manager
+
+<!-- **See all the available features in the API documentation ↗** [link to API documentation] -->
+
+- [Check for stored credentials](#check-for-stored-credentials)
+- [Retrieve stored credentials](#retrieve-stored-credentials)
```

Comment:
Not referencing storing and clearing credentials, because again as of now the `CredentialsManager` abstract class is not exported, thus there is no actual support for custom Credentials Manager.

Reply:
Also not referencing creating an instance of the `DefaultCredentialsManager`, because it is not exported, and even if it were, it depends on types used internally (`Account` and `UserAgent`). Therefore, it is not usable as of now. Meaning that, the way things stand now, developers that won't be using Web Auth will need to implement their own storage solution.